governance/votes/Executive vote - November 26, 2021.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Template - [Executive Vote] Onboarding New Vault Types, Parameter Changes, and Core Unit Budget Distributions - November 26, 2021
+summary: Onboarding wBTC-C and PSM-GUSD-A, linearly increasing the surplus buffer, increasing D3M Debt Ceiling, and distributing Core Unit Budgets.
+date: 2021-11-26T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Onboarding New Vault Types, Parameter Changes, and Core Unit Budget Distributions - November 26, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
```

Comment:
Small thing, but the template has change and additions separately, this seems fine though tbh

Reply:
Yep, was a conscious choice. Happy with this as is.